src/Symfony/Component/Console/Helper/SymfonyQuestionHelper.php

```diff
@@ -98,9 +98,9 @@ protected function writeError(OutputInterface $output, \Exception $error)
         parent::writeError($output, $error);
     }
 
-    private function getEofShortcut(): string
+    private function getEofShortcut(OutputInterface $output): string
     {
-        if ('Windows' === \PHP_OS_FAMILY) {
+        if (!$output->isDecorated()) {
```

Comment:
we still need the check for windows (see test failures):
```suggestion
        if ('\\' === \DIRECTORY_SEPARATOR && !$output->isDecorated()) {
```